src/dotnet/APIView/APIView/TreeRendererAPIV.cs

```diff
@@ -13,12 +14,12 @@ private static void AppendIndentsText(StringBuilder builder, int indents)
             string indent = new string(' ', indents * indentSize);
             builder.Append(indent);
         }
-        /*
+        
         private static void AppendIndentsHTML(StringBuilder builder, int indents)
         {
-            string indent = new string("<span class=\"indent\"><\\span>", indents);
+            builder.Append(Enumerable.Repeat("<span class=\"indent\"><\\span>", indents));
```

Comment:
Is this method used?

Reply:
Not at the moment - using the existing plain text representation of a model in a <pre> HTML tag properly formats the code in HTML, so manual HTML rendering hasn't been incorporated yet. One of the next steps in the project is to add keyword coloring though, so this will probably change.